lib/master.js

```diff
@@ -68,14 +68,21 @@ function status() {
 }
 
 // TODO(schoon) - Check for valid size with `typeof`, et al or use `is2`.
-function setSize(size) {
+// cluster-control will set the size up and down in the process of restarting
+// the cluster. During this time, we want the 'apparent size' of the cluster to
```

Comment:
More like "desired size" or "configured size" isn't it? I guess by lying about the size we sort of fulfill this claim that it is the "apparent" size (just don't count the actually processes in `slc ctl status`, right?).